docs/overview_and_guide.rst

```diff
@@ -227,20 +237,84 @@ After account creation, launch Raiden with the path of your keystore supplied::
 
 - **Using Infura**
 
+Sign up with `Infura <https://infura.io/>`_ to get an API token. After that you can start using Raiden directly::
 
-In order to use Raiden with an rpc-endpoint provided by an Infura Ethereum node, sign up with `Infura <https://infura.io/>`_ to get an API token. After that you can start using Raiden on Ropsten directly::
+    raiden --keystore-path  ~/.ethereum/testnet/keystore --eth-rpc-endpoint "https://<network>.infura.io/v3/<yourToken>"
 
-    raiden --keystore-path  ~/.ethereum/testnet/keystore --eth-rpc-endpoint "https://ropsten.infura.io/v3/<yourToken>" --pathfinding-service-address $PFS_ADDRESS
-
-.. note::
-    When you want to use a testnet you need to update the URL of the infura endpoints, e.g. for the ropsten testnet use ``https://ropsten.infura.io/v3/<yourToken>``
+Where `<network>` can be mainnet, ropsten, etc.
 
 Select the desired Ethereum account when prompted, and type in the account's password.
 
-3. The Pathfinding Address
-***************************
+3. Depositing tokens to pay the services
+****************************************
+
+To pay the services, you have to lock some of your Raiden tokens in the ``UserDeposit`` contract.
+Normally the Raiden Wizard or some other auxilliary scripts would do the contract interaction for you.
+In this section, we will briefly explain how it can be done manually so you are able to use Raiden without them.
+
+All services that are registered in the service registry of a given network will use one shared instance of ``UserDeposit``.
+You can obtain the address of the contract from
+
+    ``https://github.com/raiden-network/raiden-contracts/blob/master/raiden_contracts/data/deployment_services_<network>.json``
+
+where ``network`` is one of ``mainnet``, ``ropsten``, ``rinkeby`` or ``goerli``.
+
+Log into MyEtherWallet, the geth console or any other Ethereum interface you like with an account
+that holds enough Raiden tokens, and call ``UserDeposit.deposit(<beneficiary>, <total_deposit>)``.
```

Comment:
Don't we need to call `approve` on the token here before calling `deposit`? @jomuel